src/components/ui/tooltip/tooltip.scss

```diff
@@ -19,6 +19,7 @@ $triangle-size: 10px;
   &--visible {
     visibility: visible;
     opacity: 1;
+    transition-delay: 2s;
```

Comment:
I think we'll need to add the delay to the [base class](https://github.com/kedro-org/kedro-viz/blob/dc068db7e43d5abe84618085ea567cca6e71008e/src/components/ui/tooltip/tooltip.scss#L16). In testing, if it's only added to the modified class, there's some weirdness when the tooltip text changes and moves to other nodes. This is because the toolip itself is only ever on DOM element on the page and isn't dynamically created.

Reply:
@cvetankanechevska I think you still need to make this change. This delay needs to go on the base class of `.pipeline-tooltip`, otherwise the user could see a weird in between state of the tooltip, as seen here:

<img width="658" alt="Screen Shot 2022-06-13 at 13 14 21" src="https://user-images.githubusercontent.com/16869061/173351587-0c7acb32-a3cb-4fec-ba81-93f77dfa3bff.png">